frontend/src/_styles/theme.scss

```diff
@@ -2077,13 +2077,8 @@ input:focus-visible {
 
 input[type=text] {
   border-radius: 0 !important;
-  outline-color: #dadcde !important;
```

Comment:
need this for removing blue outline from certain i/p fields

Reply:
You folks should avoid putting `!important` wherever possible.